site/content/flowcharts/high-level.md

```diff
@@ -1,27 +1,30 @@
-## Single Developer Workflow
+## Code Review Workflow
 
-Here's a high-level view of a workflow you can follow as
-a single developer that would like to make code review
-part of your development process.
+Here's a high-level view of a workflow researchers can follow to make code
+review part of their development process.
```

Comment:
I think we agreed we were ditching language about a single/solo/lone coder at this top level flowchart. Please correct me if wrong.